core/pom.xml

```diff
@@ -60,6 +60,12 @@
       <version>${log4j.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
This is here to quiet down some warnings we were seeing (https://www.slf4j.org/codes.html#StaticLoggerBinder).